tagstudio/src/qt/modals/tag_database.py

```diff
@@ -92,8 +92,9 @@ def update_tags(self, query:str):
 				l = QHBoxLayout(c)
 				l.setContentsMargins(0,0,0,0)
 				l.setSpacing(3)
-				tw = TagWidget(self.lib, self.lib.get_tag(tag_id), True, False)
+				tw = TagWidget(self.lib, self.lib.get_tag(tag_id), True, False, True)
```

Comment:
Should probably remove the ability to delete "System Tags" (archived & favorite) since TagStudio recreates those tags on system shutdown if they are missing. 
```suggestion
				if tag_id in [0, 1]:  # [0, 1] should probably be extracted as a constant during refactor
					tag_deletable = False
				else:
					tag_deletable = True
				tw = TagWidget(self.lib, self.lib.get_tag(tag_id), True, False, tag_deletable )
```